test/safeMath.js

```diff
@@ -48,4 +50,33 @@ contract("SafeMath", () => {
         assert.equal(Number(await locker.calculateInterest(56713, 10000)), 568);
         assert.equal(Number(await locker.calculateInterest(120000, 10000)), 1200);
     });
+
+    it("should round properly when using roundedDiv", async function() {
```

Comment:
maybe add a case for throw on division by zero

Reply:
added a test case for div by zero